<launch>
    <arg name="use_game_settings" default="false"/>

    <rosparam file="$(find humanoid_league_game_controller)/config/game_controller.yaml" command="load"/>

    <node name="team_communication" pkg="humanoid_league_team_communication" type="humanoid_league_team_communication.py"  output="screen">
        <rosparam file="$(find humanoid_league_team_communication)/config/team_communication_config.yaml" command="load"/>
    </node>

    <group if="$(arg use_game_settings)">
            <rosparam command="load" file="$(find bitbots_bringup)/config/game_settings.yaml" />
    </group>
</launch>
